### PR TITLE
Feature/new document

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
Moved New Document feature to ECMS.
This PR should be merged after approving and merging https://github.com/exodev/ecms/pull/510

### CODE REVIEW

File: services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineNewDocumentTemplatePlugin.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+   * @param parent the parent
+   * @param title the title
+   * @param template the template
+   * @return the node
+   * @throws Exception the exception
+   */
+  @Override
+  public Node createDocument(Node parent, String title, DocumentTemplate template) throws Exception {
+    LOG.debug("Creating new document {} from template {}", title, template);
+    NewDocumentService documentService = ExoContainerContext.getCurrentContainer()
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineNewDocumentTemplatePlugin.java]: NewDocumentService can be injected via constructor dependency by the container.

File: services/src/main/resources/locale/officeonline/OfficeonlineClient_en.xml [hunk 2]
```diff
@@ -4,7 +4,6 @@
   <UIActionBar>
     <tooltip>
       <OfficeOnlineOpen>Edit in MS</OfficeOnlineOpen>
```
Comment (pnedonosko) [inline services/src/main/resources/locale/officeonline/OfficeonlineClient_en.xml]: Edit in Office Online

**Review by pnedonosko — CHANGES_REQUESTED** [review]

File: services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineNewDocumentTemplatePlugin.java (hunk 1 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineNewDocumentTemplatePlugin.java]: Fixed

File: services/src/main/resources/locale/officeonline/OfficeonlineClient_en.xml (hunk 2 above)
Comment (VladMartyshevskyi) [reply services/src/main/resources/locale/officeonline/OfficeonlineClient_en.xml]: Fixed

**Review by pnedonosko — APPROVED** [review]

**Pull request merged**